src/request-pipeline/context.ts

```diff
@@ -381,11 +384,16 @@ export default class RequestPipelineContext {
         if (this.isHTMLPage && this.session.disablePageCaching)
             headerTransforms.setupPreventCachingHeaders(headers);
 
+        logger.proxy('Proxy response %s %d %j', this.requestId, this.destRes.statusCode, headers);
+
         res.writeHead(this.destRes.statusCode as number, headers);
         res.addTrailers(this.destRes.trailers as http.OutgoingHttpHeaders);
     }
 
     async mockResponse (): Promise<void> {
+        // @ts-ignore
+        logger.destination('Destination request is mocking %s %s %j', this.requestId, this.mock.statusCode, this.mock.headers);
```

Comment:
Make the `ResponseMock.statusCode` and `ResponseMock.headers` properties public and remove `ts-ignore` comment.